README.md

```diff
@@ -62,13 +62,12 @@ _Note: `4.5.1` can be replaced by any [released](https://github.com/pixijs/pixi.
 
 ### Demos ###
 
-- [WebGL Filters!](http://pixijs.github.io/pixi-filters/examples/)
-- [Run pixie run](http://www.goodboydigital.com/runpixierun)
+- [Filters Demo](http://pixijs.io/pixi-filters/tools/demo/)
+- [Run Pixie Run](http://www.goodboydigital.com/runpixierun)
 - [Fight for Everyone](http://www.goodboydigital.com/casestudies/fightforeveryone)
 - [Flash vs HTML](http://flashvhtml.com)
-- [Bunny Demo](http://www.goodboydigital.com/pixijs/bunnymark)
+- [Bunny Demo](http://pixijs.io/bunny-mark)
```

Comment:
Howdy! I think we should keep the 'http://www.goodboydigital.com/pixijs/bunnymark' link here as its a little more aspirational! (and working now!)

The other bunny mark is more of a practical tool for use to test perf on various branches.

Other than that, all good!

Reply:
Sounds good!